.github/workflows/hermetic_library_generation.yaml

```diff
@@ -16,13 +16,15 @@
 name: Hermetic library generation upon generation config change through pull requests
 on:
   pull_request:
-    paths:
-      - 'generation_config.yaml'
```

Comment:
Can you put these lines back? They were added in https://github.com/googleapis/java-storage/commit/34e99b1f9bdc18a65521e670385763c70561ee86

Reply:
Restored